src/control/XournalMain.cpp

```diff
@@ -152,23 +145,23 @@ void XournalMain::checkForErrorlog() {
 
     int res = gtk_dialog_run(GTK_DIALOG(dialog));
 
-    Path errorlogPath = Util::getCacheSubfolder(ERRORLOG_DIR);
+    auto errorlogPath = Util::getCacheSubfolder(ERRORLOG_DIR);
     errorlogPath /= errorList[0];
     if (res == 1)  // Send Bugreport
     {
-        Util::openFileWithDefaultApplicaion(PROJECT_BUGREPORT);
-        Util::openFileWithDefaultApplicaion(errorlogPath);
+        Util::openFileWithDefaultApplication(PROJECT_BUGREPORT);
+        Util::openFileWithDefaultApplication(errorlogPath);
     } else if (res == 2)  // Open Logfile
     {
-        Util::openFileWithDefaultApplicaion(errorlogPath);
+        Util::openFileWithDefaultApplication(errorlogPath);
```

Comment:
I briefly fixed this typo when moving this function into PathUtil. Can undo if wanted.